packages/core/src/hooks/multiChainCalls.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { MockProvider } from '@ethereum-waffle/provider'
+import { Contract } from '@ethersproject/contracts'
+import { useChainCall } from '..'
+import { expect } from 'chai'
+import { renderWeb3Hook, deployMockToken, MOCK_TOKEN_INITIAL_BALANCE } from '../testing'
+import { encodeCallData } from '../helpers'
+import { ChainId } from '../constants/chainId'
+
+const secondTestChainId = 2137
+
+describe.only('multiChainCalls', () => {
```

Comment:
I would move those tests into the test suites for specific hook. For example this would be in `useChainCall.test.ts`